src/sentry/models/integration.py

```diff
@@ -13,10 +13,26 @@
 from sentry.signals import integration_added
 
 
+class PagerDutyService(Model):
+    __core__ = False
+
+    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
+    integration_key = models.CharField(max_length=255)
+    service_id = models.CharField(max_length=255)
+    service_name = models.CharField(max_length=255)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_pagerdutyservice"
+        unique_together = (("service_id", "integration_key"),)
```

Comment:
@MeredithAnya @wedamija  In PagerDuty, can the `integration_key` change for a given service? And should we prevent multiple Sentry orgs from integrating with a single PagerDuty org? I think it might be safer to put the `organization_integration` in the `unique_together` because I think we shouldn't necessarily block that case.

Reply:
> In PagerDuty, can the integration_key change for a given service?

I don't think so.

> And should we prevent multiple Sentry orgs from integrating with a single PagerDuty org? 

I think we were going to allow people to integration with multiple sentry organizations if they need